### PR TITLE
add left side bar design

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the Left Sidebar design

In this PR, **537 additions and 53 deletions** lines of code
- some files contain only 5 to 6 lines of code
- if you don't like, you can avoid reviewing BACKEND related files

### Please review this PR and give your feedback

---

#### This is how it looks, [Live Link](https://think-loud-git-component-left-sidebar-hsnice16.vercel.app/)
image

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
Everything looks good. Nice that you are using Material UI. You can merge it.

File: src/components/left-side-navbar/LeftSideNavbar.module.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  max-width: max-content;
+  transition: background-color 0.3s;
+}
+
+.link:hover {
+  background-color: var(--COLOR-BACKGROUND-DARK-100);
+}
+
+.listText > :is(span, p) {
+  color: var(--COLOR-TEXT);
```
Comment (anik31) [inline src/components/left-side-navbar/LeftSideNavbar.module.css:39]: appreciation - came to know about a new css selector.

File: src/data/left-side-navbar-data.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+} from "utils";
+
+export const LeftSideNavbarData = {
+  logoImg: {
+    altText: "think loud logo",
+    src: logoPNG,
+  },
+
+  links: [
+    {
```
Comment (anik31) [inline src/data/left-side-navbar-data.js:27]: appreciation - good that you've kept all the sidebar data in a single object

**Review by anik31 — APPROVED** [review]
Good job Himanshu, I liked that you've used outline icons for sidebar & changing it to filled icon when item is active.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**